.circleci/config.yml

```diff
@@ -211,12 +203,20 @@ jobs:
 
     preview-docs:
         executor: node
-
         steps:
             - downstream
             - run:
-                  name: Generate Docs
-                  command: yarn docs:preview
+                  name: Generate Custom Elements Manifest
+                  command: yarn docs:analyze
+            - run:
+                  name: Move CEM to Storybook directory
+                  command: cp projects/documentation/custom-elements.json storybook/
+            - run:
+                  name: Build documentation
+                  command: yarn docs:build
+            - run:
+                  name: Build Storybook
+                  command: yarn storybook:build
```

Comment:
you have no clue how happy this makes me!! thank you for breaking the steps out and moving away from an overly complex command!